benchexec/tools/veriabs.py

```diff
@@ -0,0 +1,73 @@
+"""
+BenchExec is a framework for reliable benchmarking.
+This file is part of BenchExec.
+
+Copyright (C) 2007-2015  Dirk Beyer
+All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import logging
+import xml.etree.ElementTree as ET
+
+import benchexec.util as util
+import benchexec.tools.template
+import benchexec.result as result
+
+class Tool(benchexec.tools.template.BaseTool):
+    """
+	veriabs for SV-COMP 2017
+    """
+    def executable(self):
+        return util.find_executable('veriabs')
+
+
+    def version(self, executable):
+        return " Version 0.001 Alpha"
```

Comment:
Hard-coding a version number in this module is not a good idea. If people run a newer version of your tool with BenchExec they will get a wrong version number in the output, which could be very confusing.

The best would be to add a parameter like `--version` to your tool and read and return the actual version number here.

If this is not possible, please remove the method completely.